client/client_opts.go

```diff
@@ -243,6 +252,14 @@ func WithMaxConcurrentDownloads(max int) RemoteOpt {
 	}
 }
 
+// WithMaxConcurrentDownloads sets max concurrent download limit.
```

Comment:
The comment is still referencing WithMaxConcurrentDownloads

Reply:
Okay, thanks, should be good this time !